app/models/authentication.rb

```diff
@@ -14,6 +14,7 @@ def self.new(*args, &block)
   encrypt_column :service_account
 
   belongs_to :resource, :polymorphic => true
+  belongs_to :manager,  :class_name => "ExtManagementSystem"
```

Comment:
Why do we need this additional reference?  Isn't the polymorphic reference (`resource`) enough?